cli/src/test/resources/ide-projects/tomcat/project/software/tomcat/conf/server.xml

```diff
@@ -0,0 +1 @@
+<Connector port="49152"/>
```

Comment:
After reading the test, I understand why this file is there and what this magic number is for.
However, what confused me is that this is in a preinstalled `software/tomcat` folder.
First of all in IDEasy the `software` folder should only contain symlinks to the actual software installed to the software repository.
Second, a valid tool folder in `software` needs to have a `.ide.software.version` file (or `.devon.software.version`).
As this is not present, it should actually not be able to determine the current installed version of `tomcat` and therefore install the proper one, delete this `software/tomcat` directory and then create a symlink here pointing to the proper installation of tomcat. If that is not the case, did you find a bug with your test if it is green?
I would expect this file to be in `repository/tomcat/tomcat/default/conf/server.xml` instead of in the `software` folder.

Reply:
I changed the location of the file accordingly, and also the port number to a more understandable one, namely 8080. This is also the port in which tomcat runs by default.